src/program/lwaftr/tests/data/tunnel_icmp.conf

```diff
@@ -1,101 +1,124 @@
 softwire-config {
   binding-table {
-    psid-map {
-      addr 178.79.150.15;
-      psid-length 4;
-      shift 12;
-    }
-    psid-map {
-      addr 178.79.150.233;
-      psid-length 16;
-      shift 0;
-    }
-    psid-map {
-      addr 178.79.150.1;
-      psid-length 0;
-      shift 16;
-    }
-    psid-map {
-      addr 178.79.150.2;
-      psid-length 16;
-      shift 0;
-    }
-    psid-map {
-      addr 178.79.150.3;
-      psid-length 6;
-      shift 10;
+    softwire {
+      ipv4 178.79.150.3;
+      psid 4;
+      b4-ipv6 127:14:25:36:47:58:69:128;
+      br-address 1e:2:2:2:2:2:2:af;
+      port-set {
+        psid-length 0;
+        reserved-ports-bit-count 16;
```

Comment:
AFAIU this initial config always has psid-length+ shift = 16 so reserved-ports-bit-count should be 0; no?  This comment applies here and elsewhere.